.github/workflows/node-versions.yml

```diff
@@ -0,0 +1,66 @@
+# 
+# This special file aims to run node tests for each relevant node version. 
+# A relevant node version can be: active, current, maintenance or EOL with some months of tolerance
+# For more details, please check ./scripts/node-versions.js
+# 
+
+name: Node versions
+on:
+  schedule:
+    - cron: 0 0 * * *
+
+jobs:
+  get-node-versions:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+
+      - uses: actions/checkout@v2
+
+      - id: set-matrix
+        run: echo "::set-output name=matrix::$(npx testable-node-versions)"
```

Comment:
Attention to this line. It runs an npm script published by me. It contains the logic behind the version matching.